POGOLib.Official/Net/RpcClient.cs

```diff
@@ -63,6 +65,9 @@ public class RpcClient : IDisposable
 
         private readonly Semaphore _rpcQueueMutex = new Semaphore(1, 1);
 
+        public event EventHandler<GetHatchedEggsResponse> HatchedEggsReceived;
+        public event EventHandler<CheckAwardedBadgesResponse> CheckAwardedBadgesReceived;
```

Comment:
it is needed for https://github.com/AeonLucid/POGOLib/pull/97/files#diff-f9807ab8e1379a13294d996b27e4d204R749